.appveyor.yml

```diff
@@ -0,0 +1,124 @@
+version: 1.0.{build}
+branches:
+  only:
+  - master
+clone_folder: C:\projects\appleseed
+image: 
+  - Visual Studio 2017
+  - Visual Studio 2015
+  - Visual Studio 2013
+configuration: Debug
+platform: x64
+install:
+- cmd: >-
+    appveyor DownloadFile https://github.com/appleseedhq/appleseed-deps/releases/download/v17/appleseed-deps-stage-win64-vs141-v17.zip
+    
+    7z x appleseed-deps-stage-win64-vs141-v17.zip -oappleseed-deps-stage-win64-vs141-v17
+
+    rm appleseed-deps-stage-win64-vs141-v17.zip
+
+before_build:
+- cmd: >-
+    mkdir build
+
+    cd build
+
+    cmake -G "Visual Studio 15 2017 Win64" -DWITH_DISNEY_MATERIAL=ON -DWITH_STUDIO=OFF -DUSE_SSE42=ON -DBOOST_ROOT=C:\Libraries\boost_1_66_0 -DPYTHON_INCLUDE_DIR=C:\Python27-x64\include -DPYTHON_LIBRARY=C:\Python27-x64\libs\python27.lib -DAPPLESEED_DEPS_STAGE_DIR=C:\projects\appleseed\appleseed-deps-stage-win64-vs141-v17\stage\vc141 ..
+
+for:
+-
+  matrix:
+    only:
+      - image: Visual Studio 2015
+  install:
+  - cmd: >-
+      appveyor DownloadFile https://github.com/appleseedhq/appleseed-deps/releases/download/v17/appleseed-deps-stage-win64-vs140-v17.zip
+
+      appveyor DownloadFile https://www.dropbox.com/s/jh7tud1tovujwd3/boost_1_55_0_2015_py35.zip?dl=1
+
+      7z x appleseed-deps-stage-win64-vs140-v17.zip -oappleseed-deps-stage-win64-vs140-v17
+
+      7z x boost_1_55_0_2015_py35.zip -oboost_1_55_0_2015_py35
+
+      rm appleseed-deps-stage-win64-vs140-v17.zip
+
+      rm boost_1_55_0_2015_py35.zip
+      
+  before_build:
+  - cmd: >-
+      mkdir build
+
+      cd build
+
+      cmake -G "Visual Studio 14 2015 Win64" -DWITH_DISNEY_MATERIAL=ON -DWITH_STUDIO=OFF -DUSE_SSE42=ON -DBOOST_ROOT=C:\projects\appleseed\boost_1_55_0_2015_py35\boost_1_55_0_2015_py35 -DPYTHON_INCLUDE_DIR=C:\Python27-x64\include -DPYTHON_LIBRARY=C:\Python27-x64\libs\python27.lib -DAPPLESEED_DEPS_STAGE_DIR=C:\projects\appleseed\appleseed-deps-stage-win64-vs140-v17\stage\vc14 ..
+
+  test_script:
+  - cmd: >-
+      cd C:\projects\appleseed\sandbox\bin\v140\Debug
+
```

Comment:
Do we need all these blank lines?

Reply:
Yes, the blank line is needed for the commands to be separated properly.